bucket/imagemagick.json

```diff
@@ -19,7 +22,7 @@
         "conjure.exe",
         "convert.exe",
         "dcraw.exe",
-        "ffmpeg.exe",
+        ["ffmpeg.exe", "iffmpeg"],
```

Comment:
Another option would be to remove ffmpeg from imagick's bin list and add a `"depends": ["ffmpeg"]`? That probably the best solution. Agreed?